api/clientset/v1beta1/topology.go

```diff
@@ -129,6 +129,10 @@ func (t *topologyClient) Get(ctx context.Context, name string, opts metav1.GetOp
 }
 
 func (t *topologyClient) Create(ctx context.Context, topology *topologyv1.Topology, opts metav1.CreateOptions) (*topologyv1.Topology, error) {
+	topology.TypeMeta = metav1.TypeMeta{
+		Kind:       "Topology",
+		APIVersion: "networkop.co.uk/v1beta1",
+	}
```

Comment:
with this create does that fix one of the tests which that never got set?

Reply:
oh you're right I forgot about that, let me check

Reply:
yup, fixed the test